src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  height: 16px;
+  margin-top: 16px;
+  margin-left: 16px;
+  align-items: end;
+
+  &__container {
+    display: flex;
+  }
+
+  &__star {
+    display: flex;
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &__count {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
You need all variants of stars block( 1,2,3..). Check how you can easy made it with loop in scss)